LayoutTests/platform/ios/editing/editable-region/overflow-scroll-text-field-and-contenteditable-expected.txt

```diff
@@ -11,7 +11,7 @@
       (event region
         (rect (0,0) width=800 height=600)
       (editable region
-        (rect (10,203) width=153 height=20)
+        (rect (9,202) width=155 height=22)
```

Comment:
these changes look reasonable but maybe other people will want to weigh-in. @smfr maybe?

Reply:
I don't recall what editable regions are used for, so can't comment on whether this is significant. Follow the breadcrumbs from https://bugs.webkit.org/show_bug.cgi?id=209888